### ISSUE TITLE
Missing files for KNNSpeed audio

### ISSUE THREAD

**Issue by th3v0ice** [wmchris/DellXPS15-9550-OSX#47]
AppleALC.kext is missing from the 10.13/Post-Install/AD-kexts/Audio/VerbStub_knnspeed/ folder. Should we just use the one from the 10.12 folder or this kext is no longer needed? The script is trying to copy it to /Library/Extensions.

Adding the file AppleALC.kext from the CLOVER folder gives these errors while rebuilding the kextcache 

```
AppleALC.kext - dependency 'as.vit9696.Lilu' not found.
AppleALC.kext is missing dependencies (including anyway; dependencies may be available from elsewhere)
```

Reading from the original KNNSpeed's post, should these lines be removed

```
sudo rm -Rf /Volumes/EFI/EFI/CLOVER/kexts/10.13/AppleALC.kext
sudo cp -R AppleALC.kext /Library/Extensions/
```

**Comment by krim404** [comment]
AppleALC in the clover folder should work perfectly. Lilu is added there, too. Do you have sound? The error may be no error, just the fact that ALC is loaded before Lilu.

**Comment by th3v0ice** [comment]
I have sound but only over speakers, when I plugin the headphones, the sound is almost muted and distorted.

**Comment by krim404** [comment]
almost?

**Comment by th3v0ice** [comment]
Yeah, on full volume, you can hear something, faintly, but its distorted.

**Comment by krim404** [comment]
thats strange. tbh i didnt test my audio out (using BT headphones). 
however: without appleALC there would be no sound at all - so it's loading up.

**Comment by th3v0ice** [comment]
I have installed the VerbStub_knnspeed one, will restart later to see the results. Copied the AppleALC.kext to the Clover on EFI partition.

**Comment by krim404** [comment]
VerbStub_knnspeed is mandatory as the audio fix. if you didnt restart after installation - that might be the problem :-)

**Comment by th3v0ice** [comment]
Yeah, I was experiencing the problem prior to the audio fix installation. I installed it because of the before mentioned problem. :) Its interesting to note that the audio sleep issue was not present in the prior to the installation configuration.

**Comment by krim404** [comment]
yea, it seems like this issue has been solved with 10.13. So if it's fixed, can you close the issue?
